Objects/abstract.c

```diff
@@ -2881,7 +2881,47 @@ PyAIter_Check(PyObject *obj)
             tp->tp_as_async->am_anext != &_PyObject_NextNotImplemented);
 }
 
+static int
```

Comment:
Do you think it's worth inlining this function? it's kind of small and probably called quite often, but I'm not sure whether it's worth the cost (maybe the compiler would already do it for you...)

Reply:
I'm pretty sure the compiler will do it implicitly, especially for a PGO build, but I guess there is little harm in adding the keyword :)

Reply:
In general, I like having the keyword just to mark the intent for the reader as well (even if the compiler might decide *NOT* to do it at the end).

Reply:
I prefer to rely on PGO rather than using "static inline" for such function. IMO the compiler is smarter than me to decide when and how functions should be inlined.